### ISSUE TITLE
[Customer Query] Fix the conflict with rtMedia and BuddyPress Edit Activity plugin

### ISSUE THREAD

**Issue by kjnanda** [rtCamp/rtMedia#1575]
On the customer’s site, **BP Edit activity plugin** is active and hence clicking on **edit** link of activity comment shows the HTML content along with the plain text.

**Debugging steps I followed:**

- If rtmedia and BP edit activity plugins are active - HTML code is visible

- If rtmedia is active and BP edit activity plugin is not active - HTML code is not visible

- If rtmedia is not active and BP edit activity plugin is active - HTML code is not visible

In short HTML code is only visible when rtmedia is enabled. This the conflict with our plugin that we should fix.

------

```
Test environment 

1. WP - 5.3.2
2. BuddyPress - 5.1.2
3. rtMedia - 4.6
4. BuddyPress Edit Activity - 1.1.0
```
----
`2568681`

**Comment by Pradeep1308** [comment]
The BP edit activity plugin is no more maintained.
